src/content/upgrades/merge/eth-issuance/index.md

```diff
@@ -0,0 +1,100 @@
+
+## Pre-merge {#pre-merge}
+
+### Execution layer issuance {#el-issuance-pre-merge}
+
+Under proof-of-work, miners only interact with the execution layer, and are rewarded in the form of block rewards being the first to solve blocks. Since the [Constantinople upgrade](/history/#constantinople) in 2019 this reward has been 2 ETH per block. Miners are also rewarded for publishing [ommer](/glossary/#ommer) blocks, which are valid blocks that don't end up in the longest/canonical chain. These rewards max out at 1.75 ETH per ommer, and are _in addition to_ the reward issued from the canonical block. Mining is an economically intensive activity, which requires high levels of ETH issuance to sustain.
+
+### Consensus layer issuance {#cl-issuance-pre-merge}
+
+The [Beacon Chain](/history/#beacon-chain-genesis) has been live since 2020, which is secured by proof-of-stake validators, not miners. This chain was bootstrapped by Ethereum users depositing ETH one-way into a smart contract on Mainnet, which the Beacon Chain listens to, crediting the user with an equal amount on the new chain. Right now these validators are not processing transactions, and are essentially coming to consensus on the state of the validator pool itself.
+
+Issuance of new ETH on this chain is rewarded to validators for properly attesting to the state of the chain, and is also rewarded when a validator proposes a block. Rewards (or penalties) are calculated and distributed every epoch (6.4 minutes) based on validator performance for that time period. These rewards are _significantly_ lower than the 2 ETH being issued under proof-of-work today every ~13.5 seconds, as operating a validating node is not an economically intense activity, thus does not require or warrant as high a reward.
+
+### Pre-merge issuance breakdown {#pre-merge-issuance-breakdown}
+
+Total ETH supply: **~119,300,000 ETH** (as of Q2 2022)
+
+**Execution layer issuance:** <br/>
+
+- Estimating at 2.08 ETH per 13.3 seconds\*: **~4,930,000** ETH issued in a year
+- Currently inflating at **~4.13%** (4.93M per year / 119.3M total)
+- \*This includes the 2 ETH per canonical block, plus an average of 0.08 ETH over time from ommer blocks. Also uses 13.3 second, the baseline block time target without any influence from a [difficulty bomb](/glossary/#difficulty-bomb). ([See source](https://bitinfocharts.com/ethereum/))
+
+**Consensus layer issuance:**
+
+- Using 13,000,000 total ETH staked, the rate of ETH issuance is ~1600 ETH/day ([See source](https://ultrasound.money/))
+- Results in **~584,000** ETH issued in a year
+- Currently inflating at **~0.49%** (584K per year / 119.3M total)
+
+<InfoBanner>
+<strong>Total annual issuance rate: ~4.62%</strong> (4.13% + 0.49%)
+
+**~89.4%** of the issuance is going to miners on the execution layer (4.13 / 4.62 \* 100)
+
+**~10.6%** is being issued to stakers on the consensus layer (0.49 / 4.62 \* 100)
```

Comment:
Not sure why but this is unreadable in darkmode.

<img width="840" alt="Screenshot 2022-06-15 at 15 36 29" src="https://user-images.githubusercontent.com/62268199/173854311-bbc06578-a781-46bd-b0f6-9c5214152938.png">